src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/UpdateClassifierTestCases.java

```diff
@@ -0,0 +1,19 @@
+package org.mule.modules.watsonalchemylanguage.automation.functional;
+
+import org.junit.Test;
+import org.mule.tools.devkit.ctf.junit.AbstractTestCase;
+
+import com.admios.connector.watsonvisualrecognition.WatsonVisualRecognitionConnector;
+import com.admios.connector.watsonvisualrecognition.exceptions.VisualRecognitionException;
+
+public class UpdateClassifierTestCases extends AbstractTestCase<WatsonVisualRecognitionConnector>{
+	
+	public UpdateClassifierTestCases() {
+		super(WatsonVisualRecognitionConnector.class);
+	}
+	
+	@Test(expected=IllegalArgumentException.class)
+	public void testWithNullFile() throws VisualRecognitionException {
```

Comment:
Add another test validating the proper execution of the operation like in create classifier 
https://github.com/Admios/watson-visual-recognition-connector/blob/00e631529ca823ea312568de40472296a8f864f0/src/test/java/org/mule/modules/watsonalchemylanguage/automation/functional/CreateClassifierTestCases.java